Cargo.toml

```diff
@@ -9,9 +9,9 @@ edition = "2018"
 crate-type = [ "staticlib" ]
 
 [dependencies]
-ppoprf = { git = "https://github.com/brave/sta-rs", rev = "3ee28feee10219ccea78e3e45b6e0ae655070e70" }
+ppoprf = { git = "https://github.com/brave/sta-rs", rev = "6b27875845b9cf5f14e4565021bc453dcf90ee25" }
```

Comment:
3ee28fee is from August 9. The current revision is 6b27875845b9.

The last source change relevant here was ca0fa5f9ee111 (August 25) but for the sake of testing against other parts of the repo you might as well use the latest.

Reply:
> 3ee28fee is from August 9. The current revision is 6b27875845b9.

This commit does use 6b27875845b9. What change do you want me to make?